CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 10.3.1
+ - Feat: handle proxy => '' as if none was set [#912](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/912)
```

Comment:
I'd say this is a bug fix instead of feature, as trying to use and empty string as the actual proxy does not make sense. This way it also aligns with the patch level version bump.